src/Features/Core/Portable/MoveStaticMembers/MoveStaticMembersWithDialogCodeAction.cs

```diff
@@ -115,7 +115,7 @@ protected override async Task<IEnumerable<CodeActionOperation>> ComputeOperation
             var members = memberNodes
                 .Select(node => root.GetCurrentNode(node))
                 .WhereNotNull()
-                .SelectAsArray(node => (semanticModel.GetDeclaredSymbol(node!, cancellationToken), false));
+                .SelectAsArray(node => (semanticModel.GetDeclaredSymbol(node, cancellationToken), false));
 
             var pullMembersUpOptions = PullMembersUpOptionsBuilder.BuildPullMembersUpOptions(newType!, members);
```

Comment:
Probably this once can be removed too? You may need to change the `as` (line 103) to a cast for that.